lib/src/utils/constants.dart

```diff
@@ -106,3 +150,105 @@ enum AddressType {
   final int value;
   const AddressType(this.value);
 }
+
+enum OptionFlags {
+  tagCborSets(1);
+
+  final int value;
+  const OptionFlags(this.value);
+}
+
+enum TransactionSigningMode {
+  ordinaryTransaction(3),
+  poolRegistrationAsOwner(4),
+  poolRegistrationAsOperator(5),
+  multisigTransaction(6),
+  plutusTransaction(7);
+
+  final int value;
+  const TransactionSigningMode(this.value);
+}
+
+enum TxOutputFormat {
+  arrayLegacy(0),
+  mapBabbage(1);
+
+  final int value;
+  const TxOutputFormat(this.value);
+}
+
+enum TxOutputDestinationType {
+  thirdParty('third_party'),
+  deviceOwned('device_owned');
+
+  final String value;
+  const TxOutputDestinationType(this.value);
+}
+
+enum DatumType {
+  hash,
+  inline,
+}
+
+enum DRepType {
+  keyHash(0),
+  keyPath(100),
+  scriptHash(1),
```

Comment:
Are all of those enums in the `constants` file in the typescript project? If not, prob they should be separated at least a bit to avoid this file getting too large / too many items.